### DISCUSSION TITLE
filtr sorting

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#14904] [How to]
How do I set the Mandator filter so that users can not switch after clicking?

**Comment by Akaki-1991** [comment]
https://user-images.githubusercontent.com/75428743/124379550-cca70580-dcc8-11eb-8af8-bc7a937a9e86.mp4

**Reply by rskrzypczak** [reply]
You can't, and I really don't know why you want to set this up.
If you really need such quirks, you need to modify some files, which is developer work.

**Reply by Akaki-1991** [reply]
thanks.
